cpp/common/src/codingstandards/cpp/deviations/CodeIdentifierDeviation.qll

```diff
@@ -336,26 +341,60 @@ class CodeIdentifierDeviation extends TCodeIndentifierDeviation {
     )
   }
 
+  predicate hasLocationInfo(
```

Comment:
May want to a comment that this is the location of the suppressed range rather than the location of the comment itself.